### PR TITLE
update installation of fbsimctl

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

[update]
Install idb for appium-idb instead of `fbsimctl`

----

https://github.com/facebook/idb has been renamed and changed the structure.
It requires to use Python 3.6. But we need Python 2.7 to build heapdump in nodejs.
I haven't ensured the new one also can call `fbsimctl`, too.

This PR is to guide users to install previous `fbsimctl` which is tagged as `1.0.0` in the repo instead of installing it via homebrew.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/drivers/ios-xcuitest.md [hunk 1]
```diff
@@ -129,10 +129,17 @@ following desired capabilities:
 such as: biometrics, geolocation setting and window focussing.
 
 ```
-# Get the Facebook Tap.
-brew tap facebook/fb
```
Comment (mykola-mokhnach) [inline docs/en/drivers/ios-xcuitest.md]: is the formula removed from brew?

Comment (KazuCocoa) [reply docs/en/drivers/ios-xcuitest.md]: No, but it only provides 0.4.0 and HEAD for now.
The 0.4.0 is old. HEAD fails to build, today.

I was able to checkout out v1.0.6 tag from the repo and got `/bin/fbsimctl`. But not sure the command will remain so far, sine commands built by `build.sh` decreased than `v1.0.0` tag.

Thus, so far, getting the repo, checkout `v1.0.0` and built binaries are safer way.

**Comment by KazuCocoa** [comment]
note:
https://www.fbidb.io/docs/commands#set-a-simulators-location

File: docs/en/drivers/ios-xcuitest.md [hunk 2]
```diff
@@ -125,15 +125,9 @@ following desired capabilities:
 
 ### Optional Setup
 
-* Install FBSimulatorControl for better handling of various iOS Simulator operations,
+* Install idb for better handling of various iOS Simulator operations,
```
Comment (mykola-mokhnach) [inline docs/en/drivers/ios-xcuitest.md]: It probably makes sense to mention that idb is only supported since 1.13.1

**Review by imurchie — APPROVED** [review]

File: docs/en/drivers/ios-xcuitest.md (hunk 2 above)
Comment (KazuCocoa) [reply docs/en/drivers/ios-xcuitest.md]: 1.13.1 has not been released yet. 1.14.0-beta was released.
I would like to update either 1.13.1 or 1.14.0 here after the version has been fixed

**Pull request merged**